File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLGaugeChartProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLGaugeChartProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> Data;
        public String Title;
        public String TitleTextColor;
        public String TitleTextHeight;
        public String TitleTextFontString;
        public String H;
        public String CenterX;
        public String CenterY;
        public String GaugeRadius;
        public String GaugeLabelTextColor;
        public String GaugeLabelTextHeight;
        public String GaugeLabelTextFontString;
        public String AlreadyUnregisteredAnimation;
        public Object Tag;
        public String TabStopIndex;

        CCLGaugeChartProps()
        {
            Data = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLGaugeChartProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLGaugeChartProps
    {
        public class GaugeData
        {
            public float MinimumValue { get; set; }
            public float MaximumValue { get; set; }
            public Range First { get; set; }
            public Range Second { get; set; }
            public Range Third { get; set; }
            public int NumberOfMajorDivisions { get; set; }
            public int NumberOfMinorDivisions { get; set; }
            public float Value { get; set; }
        }

        public class Range
        {
            public float StartValue { get; set; }
            public float EndValue { get; set; }
            public string Color { get; set; }
        }

        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public GaugeData Data { get; set; }
        public string Title { get; set; }
        public string TitleTextColor { get; set; }
        public int TitleTextHeight { get; set; }
        public string TitleTextFontString { get; set; }
        public int H { get; set; }
        public int CenterX { get; set; }
        public int CenterY { get; set; }
        public int GaugeRadius { get; set; }
        public string GaugeLabelTextColor { get; set; }
        public int GaugeLabelTextHeight { get; set; }
        public string GaugeLabelTextFontString { get; set; }
        public int AlreadyUnregisteredAnimation { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }
    }
}